buildroot/share/PlatformIO/boards/malyanM200v2.json

```diff
@@ -1,10 +1,11 @@
 {
   "build": {
     "cpu": "cortex-m0",
-    "extra_flags": "-DSTM32F070xB",
+    "extra_flags": "-DNO_UNUSED -DSTM32F070xB",
     "f_cpu": "48000000L",
     "mcu": "stm32f070rbt6",
-    "genericvariant" : "MALYAN_M200_V2",
+    "genericvariant" : "MALYANM200_F070CB",
```

Comment:
"genericvariant" shouldn't mean anything in this Arduino core - it's left over from me adding PIO support for STM32GENERIC, last time I checked.

Reply:
I removed the file entirely in my commit, its not necessary.